src/shared/components/doubleHandleSlider/DoubleHandleSlider.tsx

```diff
@@ -0,0 +1,305 @@
+import * as React from 'react';
+import './styles.scss';
+import { action, computed, makeObservable } from 'mobx';
+import { inputBoxChangeTimeoutEvent } from '../../lib/EventUtils';
+
+export interface IDoubleHandleSliderProps {
+    id: string;
+    min: string;
+    max: string;
+    callbackLowerValue: (lowerValue: number) => void;
+    callbackUpperValue: (upperValue: number) => void;
+}
+
+export interface IDoubleHandleSliderState {
+    lowerValue: number;
+    upperValue: number;
+}
+
+const WIDTH = '135px';
+export default class DoubleHandleSlider extends React.Component<
```

Comment:
The two slider components which are already installed are `react-bootstrap-slider` and `react-rangeslider`. As far as I can tell, the former DOES support multiple values whereas the latter DOES NOT. Unfortunately, I had tried to get `react-bootstrap-slider` actually imported and visible in the past without luck, and all the existing code uses `react-rangeslider`. This is why I ended up creating my own component, since I didn't want to add anymore package dependencies.

Additionally, most of the code in my file is dedicated to syncing the slider with the two input boxes and having everything be the right size. Importing a double-slider wouldn't actually save that much code space.